apps/test/unit/lib/tools/jsinterpreter/CustomMarshalingInterpreterTest.js

```diff
@@ -802,9 +802,7 @@ describe('The CustomMarshalingInterpreter', () => {
       });
 
       it('will skip the inherits and trigger properties if present', () => {
-        expect(nativeFunc.inherits).to.be.defined;
```

Comment:
my original goal was to update this assertion, but when i changed `.to.be.defined` to `.to.not.be.undefined`, the test failed consistently.

the original assertion also wasn't working -- logging `nativeFunc.inherits` and `nativeFunc.trigger` always returned `undefined`, which makes sense when you look at the setup for `nativeFunc`:

https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/lib/tools/jsinterpreter/CustomMarshalingInterpreterTest.js#L774-L788

`expect(undefined).to.be.defined` passes, which tells me these assertions are useless (likely why `.defined` has been removed from chai, but i can't find any good documentation around that change). the other assertions in this test cover the behavior we want to see, so i think it's okay to just remove these assertions.